.github/workflows/cherryPick.yml

```diff
@@ -100,20 +100,21 @@ jobs:
           USER: OSBotify
           TITLE_REGEX: Update version to ${{ env.NEW_VERSION }}
 
-      - name: Cherry-pick the merge commits to new branch
-        id: cherryPick
+      - name: Cherry-pick the version-bump to new branch
         run: |
           git fetch
-          git cherry-pick ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }} ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }} --mainline 1
-        continue-on-error: true
+          git cherry-pick -x --mainline 1 --strategy=recursive -Xtheirs ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }}
 
-      # If there is a merge conflict, we'll just commit what we have,
-      # and the PR will be auto-assigned for someone to manually resolve the conflicts.
-      - name: If there is a merge conflict...
-        if: ${{ steps.cherryPick.outcome == 'failure' }}
+      - name: Cherry-pick the merge commit of target PR to new branch
+        id: cherryPick
         run: |
-          git add -A
-          git cherry-pick --continue
+          if [[ "$(./.github/scripts/cherryPick.sh ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }})" ]]; then
```

Comment:
Did we test this line somehow? It looks like there is a few gotchas that could break this.

Reply:
No, we didn't test it. I can try to do something similar in `Public-Test-Repo` to validate this a bit.

Reply:
Did some testing [here](https://github.com/Andrew-Test-Org/Public-Test-Repo/pull/52/checks?check_run_id=2740976456), found one potential issue/simplification with the syntax here.